app/src/main/java/com/jerboa/datatypes/types/Others.kt

```diff
@@ -20,64 +20,64 @@ enum class SortType {
     /**
      * Posts sorted by the most recent comment.
      */
-    @SerializedName("Active")
+    @SerializedName("Active", alternate = ["0"])
```

Comment:
Don't include this, we don't want to start doing the work of supporting multiple versions by hacks in a single codebase.

Reply:
I know, but these are going to get out of hand *very quickly* with breaking changes, especially with lemmy API not even having a 1.0 release yet. So for now I'd rather not do hacks to support beta versions.